src/GoTrueApi.ts

```diff
@@ -124,14 +130,16 @@ export default class GoTrueApi {
    * Gets the user details.
    * @param jwt A valid, logged-in JWT.
    */
-  async getUser(jwt: string) {
+  async getUser(
+    jwt: string
+  ): Promise<{ user: User | null; data: User | null; error: Error | null }> {
     try {
       let headers = { ...this.headers }
       headers['Authorization'] = `Bearer ${jwt}`
       let data: any = await get(`${this.url}/user`, { headers })
-      return { data, error: null }
+      return { user: data, data, error: null }
```

Comment:
Hmm why do we need to return data twice here?

Reply:
@ykdojo originally we only returned `data` but from a DX it's nicer to return `user` IMO. But in order to return backwards compatible and don't break existing integrations I return both `data` and `user`.